src/components/collection/CollectionRecords.tsx

```diff
@@ -313,41 +314,42 @@ class Table extends PureComponent<TableProps> {
 }
 
 function ListActions(props) {
-  const { bid, cid, session, bucket, collection, hooks = [] } = props;
+  const { bid, cid, session, bucket, collection } = props;
   if (session.busy || collection.busy) {
     return null;
   }
-  const defaultButtons = [
-    <AdminLink
-      key="__1"
-      name="record:create"
-      params={{ bid, cid }}
-      className="btn btn-info btn-record-add"
-    >
-      Create record
-    </AdminLink>,
-    <AdminLink
-      key="__2"
-      name="record:bulk"
-      params={{ bid, cid }}
-      className="btn btn-info btn-record-bulk-add"
-    >
-      Bulk create
-    </AdminLink>,
-  ];
   return (
     <div className="list-actions">
-      {canCreateRecord(session, bucket, collection)
-        ? [...defaultButtons, ...hooks]
-        : hooks}
+      {canCreateRecord(session, bucket, collection) && (
+        <>
+          <AdminLink
+            key="__1"
+            name="record:create"
+            params={{ bid, cid }}
+            className="btn btn-info btn-record-add"
+          >
+            Create record
+          </AdminLink>
+          <AdminLink
+            key="__2"
+            name="record:bulk"
+            params={{ bid, cid }}
+            className="btn btn-info btn-record-bulk-add"
+          >
+            Bulk create
+          </AdminLink>
+        </>
+      )}
+      {/* won't render if kinto-signer not enabled or collection not configured
```

Comment:
since kinto-signer was decommissioned, we may want to mention the remote settings plugin, or just mention the `signer` capability instead

Reply:
[c713f8e](https://github.com/Kinto/kinto-admin/pull/2360/commits/c713f8e1224aadaa169b9ae8b3443dfa712c5081)